### ISSUE TITLE
waitForOutput check ignores first character 

### ISSUE THREAD

**Issue by blzsaa** [psxpaul/gradle-execfork-plugin#10]
WhatForOutput fails when the required output is in the very begining of the log

How to reproduce the problem:
```
task startecho(type: com.github.psxpaul.task.ExecFork) {
    commandLine = 'echo'
    args = [ 'asd' ]
    waitForOutput = 'asd'
}
```

when I run `./gradlew startecho --debug` it returns `checking if |sd\n| equals |asd|`

**Comment by blzsaa** [comment]
I created a pull request for the issue
#11

**Comment by psxpaul** [comment]
Thanks @blzsaa , this should be fixed in v0.1.8
